FirebasePerformance/Sources/AppActivity/FPRAppActivityTracker.m

```diff
@@ -66,12 +68,24 @@ @implementation FPRAppActivityTracker
 + (void)load {
   // This is an approximation of the app start time.
   appStartTime = [NSDate date];
+  
+  dispatch_async(dispatch_get_main_queue(), ^{
+    dispatch_async(dispatch_get_main_queue(), ^{
+      doubleDispatchTime = [NSDate date];
+    });
+  });
+  
   gAppStartCPUGaugeData = fprCollectCPUMetric();
   gAppStartMemoryGaugeData = fprCollectMemoryMetric();
   [[NSNotificationCenter defaultCenter] addObserver:self
                                            selector:@selector(windowDidBecomeVisible:)
                                                name:UIWindowDidBecomeVisibleNotification
                                              object:nil];
+  
+  [[NSNotificationCenter defaultCenter] addObserver:self
+                                           selector:@selector(applicationDidFinishLaunching:)
+                                               name:UIApplicationDidFinishLaunchingNotification
```

Comment:
Why are you comparing the double dispatch time with `UIApplicationDidFinishLaunchingNotification`? The Hub app is using the `ApplicationWillFinishLaunchingWithUserActivity`

Reply:
WillFinishLaunching is something that cannot be piggybacked by an SDK as we do not have any notifications in that aspect. So, we are using the closest, which is applicationDidFinishLaunchingWithOptions.